vm/ubpf_jit_arm64.c

```diff
@@ -338,7 +338,14 @@ static void
 emit_unconditionalbranch_immediate(
     struct jit_state* state, enum UnconditionalBranchImmediateOpcode op, int32_t target_pc)
 {
-    emit_patchable_relative(state->offset, target_pc, 0, state->jumps, state->num_jumps++);
+    struct patchable_relative* table = state->jumps;
+    int* num_jumps = &state->num_jumps;
+    if (op == UBR_BL && target_pc != TARGET_PC_ENTER) {
```

Comment:
Just wanted to confirm that I am reading this correctly: We set `table` and `num_jumps` assuming that this is not a local call. Then, if it _is_, we point them to the table and counter for the local call. After that, we can do the same operation in both cases. 

If that's right, then I really like it!